Tools/setup/ubuntu.sh

```diff
@@ -163,29 +170,51 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 	# arm-none-eabi-gcc
 	NUTTX_GCC_VERSION="9-2020-q2-update"
 	NUTTX_GCC_VERSION_SHORT="9-2020q2"
+	echo
+	echo $VERBOSE_BAR
+	echo "🍻 Verifying proper gcc version (${NUTTX_GCC_VERSION}), and installing if not found"
+	echo
 
-	source $HOME/.profile # load changed path for the case the script is reran before relogin
+	source $HOME/.profile
 	if [ $(which arm-none-eabi-gcc) ]; then
 		GCC_VER_STR=$(arm-none-eabi-gcc --version)
 		GCC_FOUND_VER=$(echo $GCC_VER_STR | grep -c "${NUTTX_GCC_VERSION}" || true)
 	fi
 
 	if [[ "$GCC_FOUND_VER" == "1" ]]; then
-		echo "arm-none-eabi-gcc-${NUTTX_GCC_VERSION} found, skipping installation"
+		# echo "arm-none-eabi-gcc-${NUTTX_GCC_VERSION} found, skipping installation"
+		echo "📌 Skipping installation, the arm cross compiler was found"
+		echo $VERBOSE_BAR
+		echo
 
 	else
-		echo "Installing arm-none-eabi-gcc-${NUTTX_GCC_VERSION}";
-		wget -O /tmp/gcc-arm-none-eabi-${NUTTX_GCC_VERSION}-linux.tar.bz2 https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/${NUTTX_GCC_VERSION_SHORT}/gcc-arm-none-eabi-${NUTTX_GCC_VERSION}-${INSTALL_ARCH}-linux.tar.bz2 && \
+		# echo "Installing arm-none-eabi-gcc-${NUTTX_GCC_VERSION}";
+		echo "📌 The arm cross compiler was not found";
+		echo " * Installing arm-none-eabi-gcc-${NUTTX_GCC_VERSION}";
+		COMPILER_NAME="gcc-arm-none-eabi-${NUTTX_GCC_VERSION}"
+		COMPILER_PATH="/tmp/$COMPILER_NAME-linux.tar.bz2"
+		wget -O /tmp/gcc-arm-none-eabi-${NUTTX_GCC_VERSION}-linux.tar.bz2 https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/${NUTTX_GCC_VERSION_SHORT}/gcc-arm-none-eabi-${NUTTX_GCC_VERSION}-${INSTALL_ARCH}-linux.tar.bz2
```

Comment:
We have to decide whether we want the arm-none-eabi toolchain shipped with 24.04, or the one downloaded manually. Right now you have both here.